web/src/composables/useLogs.ts

```
@@ -1253,7 +1255,7 @@ const useLogs = () => {
                 "Error while processing partition request.";
               if (err.response != undefined) {
                 searchObj.data.errorMsg = err.response.data.error;
-                if(err.response.data.hasOwnProperty("error_detail")){
+                if (err.response.data.hasOwnProperty("error_detail")) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)